templates/flutter-dio/services/service.dart.hbs

```diff
@@ -25,7 +25,7 @@ class {{title}}Service {
     }
     options.responseType = ResponseType.json;
     options.contentType = ContentType.json.toString();
-    options.headers['Accept'] = 'application/ld+json';
+    options.headers?['Accept'] = 'application/ld+json';
```

Comment:
Maybe instead of this we should always initialize the headers portion, because we always need to send this header to the backend